back/app/models/activity.rb

```diff
@@ -27,8 +27,29 @@
 #  fk_rails_...  (user_id => users.id)
 #
 class Activity < ApplicationRecord
+  MANAGEMENT_FILTERS = [
+    { item_type: 'Idea', actions: %w[created changed deleted] },
+    { item_type: 'Phase', actions: %w[created changed deleted] },
+    { item_type: 'Project', actions: %w[created changed deleted] },
+    { item_type: 'ProjectFolders::Folder', actions: %w[created changed deleted] }
+  ].freeze
+
   belongs_to :user, optional: true
   belongs_to :item, polymorphic: true, optional: true
 
   validates :action, :item_type, :item_id, presence: true
+
+  scope :management, lambda {
+    activities = where(acted_at: 30.days.ago..Time.now).where(user: User.admin_or_moderator)
```

Comment:
Does this work? Seems odd to have `..` in here

Reply:
Yes, it works, and is tested. It's a [range](https://ruby-doc.org/core-2.5.1/Range.html) thing.